source/mei-source.xml

```diff
@@ -88,7 +88,7 @@
                <p>
                   <hi>Music Encoding Initiative (MEI)</hi>
                </p>
-               <p>NOTICE: Copyright (c) 2017–2023 by the Music Encoding Initiative (MEI) Council.</p>
+               <p>NOTICE: Copyright (c) by the Music Encoding Initiative (MEI) Council.</p>
```

Comment:
Should we also take this opportunity to assign these to the MEI Board, as we have in the files below?